spec/spec_helper.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+SRC_PATH = File.expand_path("../src", __dir__)
+DATA_PATH = File.expand_path("data", __dir__)
+TEST_PATH = File.expand_path(__dir__)
+ENV["Y2DIR"] = SRC_PATH
+
+
+# Ensure the tests runs with english locales
+ENV["LC_ALL"] = "en_US.UTF-8"
+ENV["LANG"] = "en_US.UTF-8"
+
+# load it early, so other stuffs are not ignored
+if ENV["COVERAGE"]
```

Comment:
Yay!

If we also add `/coverage/` to top level `.gitignore`, the CI check should go green again

Reply:
Done